### ISSUE TITLE
Taskprovider cannot be turned on in Rider

### ISSUE THREAD

**Issue by zdomokos** [mmm444/ijphab#2]
The task provider switch stays off, cannot turn it on using Rider 2017.2

**Comment by mmm444** [comment]
Is there any exception being thrown? If it is then the exclamation mark in the lower right corner of the IDE should blink and clicking on it should show more details.

**Comment by zdomokos** [comment]
Here are the steps: 
Help -> Find Action -> type "Phabricator"
![image](https://user-images.githubusercontent.com/2656974/32204177-258e0c30-bdbe-11e7-99d4-defe963dff27.png)
I click on the off to turn on the task provider.
Rider show message box: "Restart Rider to activate changes in plugins?" clicked Restart
Rider reloads, Find action -> type "Phabricator" and I see the same thin. Phabricator is off.
I cannot see exception. I cannot see any settings added to Rider settings, provider is not listed in available servers. 
![image](https://user-images.githubusercontent.com/2656974/32204281-b37da78a-bdbe-11e7-9fe9-ad88101dabec.png)

**Comment by mmm444** [comment]
Thanks for the detailed report. I have installed Rider on my computer and now I see the cause. I have messed up the dependencies section in `plugin.xml`. The [docs](https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html) say that there must be at least one module dependency (starting with `com.intellij.modules`) to make the plugin work in non-IDEA IDEs. I have tried adding this locally and the plugin started working in Rider.

Expect the fix shortly.
